addons/web/static/src/views/list/list_renderer.scss

```
@@ -89,12 +90,46 @@
                 @include media-only(print) {
                     font-weight: $font-weight-bold;
                 }
+
+                &.table-active {
+                    background: var(--ListRenderer-thead-bg-active-color);
+                    box-shadow: none;
+                }
             }
 
             .o_list_number_th {
                 /*rtl:ignore*/
                 text-align: right;
             }
+
+            .o_column_sortable {
+
+                .o_list_sortable_caret {
+                    background: var(--ListRenderer-thead-bg-color);
+                    position: absolute;
+                    right: .2rem;
+                    padding: 0 .2rem;
+                }
+
+                &.table-active .o_list_sortable_caret {
+                    background: var(--ListRenderer-thead-bg-active-color);
+                }
+
+                .o_list_header_label_spacer {
+                    width: 0;
+                    flex-grow: 0;
+                }
+
+                &.table-active .o_list_header_label_spacer {
+                    width: 1.2rem;
+                    max-width: 1.2rem;
+                    min-width: .4rem;
+
+                    flex-shrink: 0;
+                    flex-grow: 1;
+                    flex-basis: 0;
+                }
```

I think there is some redundancy here between the `width`, `min-width`, `flex-basis`...

Here's my suggestion :
```suggestion
                .o_list_header_label_spacer {
                    flex: 0 0 0;
                }

                &.table-active .o_list_header_label_spacer {
                    flex: 1 0 0.4rem;
                    max-width: 1.2rem;
                }
```

In a nutshell, it means:
- in general, size of `0` and can neither shrink nor grow
- when the column is "active", base size is `0.4rem` and cannot shrink but is able to grow up to `1.2rem`

Note that there is a shorthand version for setting `flex-{grow,shrink,basis}` in one rule (cf. https://developer.mozilla.org/en-US/docs/Web/CSS/flex).